### ISSUE TITLE
train_metric smaller than val_metric

### ISSUE THREAD

**Issue by ruizhecao96** [sweetcocoa/DeepComplexUNetPyTorch#7]
Hallo,

I have run this code on same dataset (DEMAND+VoiceBank). But during the training the train_metric(pesq) is much smaller than val_metric, namely, train_metirc is about 2.7 and val_metric is about 3.2.  Do you have the similar situation?

**Comment by sailorbj** [comment]
I run this code with the latest version of pypesq as metric, it returns nan for every step. Do you suffer this problem? How did you fix?

**Comment by ruizhecao96** [comment]
@sailorbj Yeah I also have that problem. Using another pesq function can solve it.

**Comment by sailorbj** [comment]
Thanks for you reply. Which pesq function works well?

**Comment by ruizhecao96** [comment]
@sailorbj It was given by my tutor, sorry for can't share it. I am not sure if he is willing to.
